dist/svg/ds6/icons.svg

```diff
@@ -130,39 +130,39 @@
     <symbol id="null-watch" viewBox="0 0 63 45"><path fill="#111820" fill-rule="nonzero" d="M61.48 23.007l-15-19.68a5.84 5.84 0 0 0-10.49 3.52v.66a2.5 2.5 0 1 0 5 0v-.66a.84.84 0 0 1 1.52-.51l11.93 15.67h-14a6.5 6.5 0 0 0-6.41 5.47 7.36 7.36 0 0 0-5.18 0 6.5 6.5 0 0 0-6.36-5.47h-14l11.98-15.67a.84.84 0 0 1 1.52.51v.66a2.5 2.5 0 0 0 5 0v-.66a5.84 5.84 0 0 0-10.49-3.54l-15 19.7a2.5 2.5 0 0 0 2 4h.49v10.5a6.51 6.51 0 0 0 6.5 6.5h10a8.51 8.51 0 0 0 8.5-8.5v-1a2.5 2.5 0 0 1 5 0v1a8.51 8.51 0 0 0 8.5 8.5h10a6.51 6.51 0 0 0 6.5-6.5v-10.5h.5a2.5 2.5 0 0 0 2-4h-.01zm-37.49 11.5a3.5 3.5 0 0 1-3.5 3.5h-10a1.5 1.5 0 0 1-1.5-1.5v-10.5h13.5a1.5 1.5 0 0 1 1.5 1.5v7zm30 5a1.5 1.5 0 0 1-1.5 1.5h-10a3.5 3.5 0 0 1-3.5-3.5v-7a1.5 1.5 0 0 1 1.5-1.5h13.5v10.5z"></path></symbol>
 
     <symbol id="icon-pause" viewBox="0 0 16 16">
-        <g fill="currentColor" fill-rule="evenodd"><path d="M0 0h6v16H0zM10 0h6v16h-6z"></path></g>
+        <g fill="#111820" fill-rule="evenodd"><path d="M0 0h6v16H0zM10 0h6v16h-6z"></path></g>
```

Comment:
We should not change anything in this file. Foreground icons need to be left alone in order to to be customizable.

Reply:
Reran build, I had changed this in src to remove all fills, but forgot to run build to copy it over to dist. 
I did change this file since there were some fills from the import, so I removed all the fills.

Reply:
@seangates Also, what is the fill="currentColor"? Is that even proper? I removed it, but I can readd

Reply:
`currentColor` is a valid property, and should be left for those that need it. If it was specified before, let's keep it, as that change could have unintended consequences where these icons are used as they will turn `black` (the default SVG color).